plugins/rbac-backend/src/service/role-manager.ts

```diff
@@ -127,6 +194,24 @@ export class BackstageRoleManager implements RoleManager {
       return true;
     }
 
+    // Create Temporary roles to compare
+    const tempRoleName = createRoleName(name1, domain);
```

Comment:
We have a code upper:
```
    if (domain.length > 0) {
      throw new Error('domain argument is not supported.');
    }
```
So I don't think that we're using domain. Also I'm not sure that we need this feature at least rigth now.